public/app/features/alerting/unified/components/mute-timings/MuteTimingsTable.tsx

```diff
@@ -265,7 +240,79 @@ function useColumns(
       });
     }
     return columns;
-  }, [alertManagerSourceName, setMuteTimingName, showActions, exportSupported, exportAllowed, openExportDrawer]);
+  }, [
+    alertManagerSourceName,
+    setMuteTimingName,
+    showActions,
+    exportSupported,
+    exportAllowed,
+    openExportDrawer,
+    styles.actionsColumn,
+  ]);
+}
+
+function isDisabled(muteTiming: MuteTimeInterval) {
+  const allTimeIntervalsDisabled = muteTiming.time_intervals.every((timeInterval) => {
+    return (
```

Comment:
Could we use `getDisabledFromDto` instead of the conditions below?

Reply:
💯